config/toml.go

```diff
@@ -138,6 +138,20 @@ priv_validator_state_file = "{{ js .BaseConfig.PrivValidatorState }}"
 # connections from an external PrivValidator process
 priv_validator_laddr = "{{ .BaseConfig.PrivValidatorListenAddr }}"
 
+# Defines which private validator protocol to use gRPC or raw
+# Raw represents a raw tcp of unix socket (no http)
+# Default is raw
+priv_validator_protocol = "{{ .BaseConfig.PrivValidatorProtocol }}"
```

Comment:
It's a bit odd to me that we use both `laddr` and `protocol` to specify the protocol. Wouldn't `grpc://` be sufficient to distinguish it from `tcp://` and `unix://`?

Reply:
One could use unix sockets with gRPC to my understanding. Do we want to deprecate unix sockets?

Reply:
I suppose that's technically true, but does anyone ever really do that? I would guess most people would use `127.0.0.1` and call it a day. UNIX sockets might be marginally faster, but if we're worried about those sorts of microoptimizations we should really just cache the key instead.

Reply:
I am fine with deprecating it. Wasn't sure what others thought so tried to keep it around.

Reply:
Worth getting others' opinions, but I think axing it is fine.

Reply:
I think combining this to a single field makes sense.

Reply:
merged into one with `grpc://` as the prefix for it